<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceBetEvent</class>
 <widget class="QStackedWidget" name="PlaceBetEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>156</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="SendCoins">
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <!-- <layout class="QGridLayout" name="gridLayout"> -->
   <layout class="QHBoxLayout" name="gridLayout">
    <property name="spacing">
     <number>12</number>
    </property>

                <item>
                 <widget class="QLabel" name="labelCoinControlAutomaticallySelected">
                  <property name="text">
                   <string>Event Details</string>
                  </property>
                  <property name="margin">
                   <number>5</number>
                  </property>
                 </widget>
                </item>

                <item>
                 <spacer name="horizontalSpacerCoinControl">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>1</height>
                   </size>
                  </property>
                 </spacer>
                </item>

                <item>
                 <widget class="QPushButton" name="pushButtonPlaceHomeBet">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Home</string>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>

                <item>
                 <widget class="QPushButton" name="pushButtonPlaceAwayBet">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Away</string>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>

                <item>
                 <widget class="QPushButton" name="pushButtonPlaceDrawBet">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Draw</string>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>



    <!-- <item row="0" column="0">
     <widget class="QLabel" name="payToLabel">
      <property name="text">
       <string>Pay &amp;To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>Event Details</cstring>
      </property>
     </widget>
    </item>

    <item row="0" column="1">
     <widget class="QLabel" name="payToLabel">
      <property name="text">
       <string>Pay &amp;To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>payTo</cstring>
      </property>
     </widget>
    </item> -->



    <!-- <item row="0" column="0">
     <widget class="QLabel" name="payToLabel">
      <property name="text">
       <string>Pay &amp;To:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>payTo</cstring>
      </property>
     </widget>
    </item>

    <item row="0" column="1">
     <layout class="QHBoxLayout" name="payToLayout">
      <property name="spacing">
       <number>0</number>
      </property>

      <item>
       <widget class="QValidatedLineEdit" name="payTo">
        <property name="toolTip">
         <string>The Wagerr address to send the payment to</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="addressBookButton">
        <property name="toolTip">
         <string>Choose previously used address</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../wagerr.qrc">
          <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+A</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="pasteButton">
        <property name="toolTip">
         <string>Paste address from clipboard</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../wagerr.qrc">
          <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QToolButton" name="deleteButton">
        <property name="toolTip">
         <string>Remove this entry</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../wagerr.qrc">
          <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item> -->



   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>bitcoinamountfield.h</header>
  </customwidget>
 </customwidgets>
 <!-- <tabstops>
  <tabstop>payTo</tabstop>
  <tabstop>addressBookButton</tabstop>
  <tabstop>pasteButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>addAsLabel</tabstop>
  <tabstop>payAmount</tabstop>
  <tabstop>payAmount_is</tabstop>
  <tabstop>deleteButton_is</tabstop>
  <tabstop>payAmount_s</tabstop>
  <tabstop>deleteButton_s</tabstop>
 </tabstops> -->
 <resources>
  <include location="../wagerr.qrc"/>
 </resources>
 <connections/>
</ui>
